From c7ff651960a6ef11cef55479658aff504c34872f Mon Sep 17 00:00:00 2001
From: Chaitanya Kulkarni <chaitanya.kulkarni@wdc.com>
Date: Sun, 21 Feb 2021 21:29:57 -0800
Subject: [PATCH] blktrace: fix blk_rq_issue documentation
Git-commit: c7ff651960a6ef11cef55479658aff504c34872f
Patch-mainline: v5.12-rc2
References: git-fixes
Modified-by-SEL: Yes, refreshed due to different context


The commit 881245dcff29 ("Add DocBook documentation for the block tracepoints.")
added the comment for blk_rq_issue() tracepoint. Remove the duplicate
word from the tracepoint documentation.

Fixes: 881245dcff29 ("Add DocBook documentation for the block tracepoints.")
Signed-off-by: Chaitanya Kulkarni <chaitanya.kulkarni@wdc.com>
Reviewed-by: Johannes Thumshirn <johannes.thumshirn@wdc.com>
Reviewed-by: Damien Le Moal <damien.lemoal@wdc.com>
Reviewed-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Jens Axboe <axboe@kernel.dk>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 include/trace/events/block.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/trace/events/block.h
+++ b/include/trace/events/block.h
@@ -199,7 +199,7 @@ DEFINE_EVENT(block_rq, block_rq_insert,
 /**
  * block_rq_issue - issue pending block IO request operation to device driver
  * @q: queue holding operation
- * @rq: block IO operation operation request
+ * @rq: block IO operation request
  *
  * Called when block operation request @rq from queue @q is sent to a
  * device driver for processing.
